app/models/bicycle.rb

```diff
@@ -1,3 +1,4 @@
 class Bicycle < ApplicationRecord
 	belongs_to :station
```

Comment:
I'm not pretty sure do we need this relation or not.
a book can belong to an author and it's a static relationship.
a bicycle change from station to station quite often, it's dynamic.